src/object/srv_obj_migrate.c

```diff
@@ -632,6 +754,10 @@ migrate_fetch_update_inline(struct migrate_one *mrone, daos_handle_t oh,
 	    !obj_shard_is_ec_parity(mrone->mo_oid, &oca))
 		mrone_recx_daos2_vos(mrone, oca);
 
+	rc = csum_init_csummer(csums_iov, &csummer);
+	if (rc != 0)
```

Comment:
D_GOTO(out, rc) to free csum_iov_fetch? And also two return above for FAIL_CHECK.

Reply:
Good catch. Will fix.